fvm/fvm_test.go

```diff
@@ -1775,3 +1776,35 @@ func TestBlockContext_ExecuteTransaction_FailingTransactions(t *testing.T) {
 			}),
 	)
 }
+func TestSigningWithTags(t *testing.T) {
+	t.Run("Signing Transactions without tag works (for now)", newVMTest().
```

Comment:
Can you add a test case for signing _with_ a tag?

Reply:
Actually my process was:
- remove ability to sign without a tag
- make sure all tests pass by adding the tag where necessary (I also used a local version of the SDK that added the tag)
- re-add ability to sign without a tag

So all other tests (that do signing) use a tag (or will use it when the SDK version is updated). That is why I only added a test for explicitly not using it.

Do you still think it makes sense to add an explicit test case for using the tag?